CMakeLists.txt

```diff
@@ -79,25 +70,25 @@ message(STATUS "Build .Net Binding: ${BUILD_DOTNET}")
 
 # Add OR Tools Dependencies as CMake subproject if missing
 if(NOT Swig_DIR AND BUILD_DEPS)
-  set(Swig_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "Swig dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/SWIG)
 endif()
 if(NOT gflags_DIR AND BUILD_DEPS)
-  set(gflags_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "gflags dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/gflags)
 endif()
 if(NOT glog_DIR AND BUILD_DEPS)
-  set(glog_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "glog dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/glog)
 endif()
 if(NOT ZLIB_DIR AND BUILD_DEPS)
-  set(ZLIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "ZLIB dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/ZLIB)
 endif()
 if(NOT Protobuf_DIR AND BUILD_DEPS)
-  set(Protobuf_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "Protobuf dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/Protobuf)
 endif()
 if(NOT abseil_DIR AND BUILD_DEPS)
-  set(abseil_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external CACHE PATH "abseil dependency path")
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/external/abseil)
```

Comment:
abseil-cpp ? (I can also do it in a later commit) 
just becaue there is also https://github.com/abseil/abseil-py and maybe we will need it one day too